app/src/main/java/org/wikipedia/page/PageFragment.kt

```diff
@@ -127,6 +128,7 @@ class PageFragment : Fragment(), BackPressedHandler, CommunicationBridge.Communi
     private lateinit var leadImagesHandler: LeadImagesHandler
     private lateinit var pageFragmentLoadState: PageFragmentLoadState
     private lateinit var bottomBarHideHandler: ViewHideHandler
+    lateinit var articleInteractionEvent: ArticleInteractionEvent
```

Comment:
Since this is a `TimedEvent`, do we need to `pause()` and `resume()` it when the PageFragment is paused and resumed?